package.json

```diff
@@ -21,10 +21,10 @@
         "arches-dev-dependencies": "archesproject/arches-dev-dependencies#dev/8.0.x"
     },
     "dependencies": {
-        "arches": "archesproject/arches#dev/8.0.x"
-    },
-    "nodeModulesPaths": {
+        "arches": "archesproject/arches#dev/8.0.x",
+        "vue-router": "^4.4.3"
```

Comment:
I don't feel very strongly about this yet, but I could see `vue-router` going into `dev/8.0.x`

Reply:
For sure, let's cross that bridge when we need to.